plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeClientModule.java

```diff
@@ -69,6 +69,8 @@ public ConnectionFactory getConnectionFactory(BaseJdbcConfig baseJdbcConfig, Sno
         properties.setProperty("TIMESTAMP_TZ_OUTPUT_FORMAT", "YYYY-MM-DD\"T\"HH24:MI:SS.FF9TZH:TZM");
         properties.setProperty("TIMESTAMP_LTZ_OUTPUT_FORMAT", "YYYY-MM-DD\"T\"HH24:MI:SS.FF9TZH:TZM");
         properties.setProperty("TIME_OUTPUT_FORMAT", "HH24:MI:SS.FF9");
+        properties.setProperty("GEOGRAPHY_OUTPUT_FORMAT", "GeoJSON");
+        properties.setProperty("GEOMETRY_OUTPUT_FORMAT", "GeoJSON");
```

Comment:
I believe GeoJSON is the default value. Revert.

Reply:
Yes it's the default value. Actually I intentionally added them so that they're pinned, (to avoid future SF changes breaking things). WDYT?